src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, num) {
+      operation(num);
+
+      return calculator;
+    },
+
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
+    subtract(num) {
+      calculator.result -= num;
+
+      return calculator;
+    },
+    multiply(num) {
+      calculator.result *= num;
+
+      return calculator;
+    },
+    divide(num) {
+      calculator.result /= num;
+
+      return calculator;
+    },
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
+    },
```

Comment:
You can return 'this' instead of calculator, but it's OK!

Reply:
➕